homeassistant/config.py

```diff
@@ -663,6 +667,14 @@ async def async_process_ha_core_config(hass: HomeAssistant, config: dict) -> Non
     if CONF_UNIT_SYSTEM in config:
         hac.units = get_unit_system(config[CONF_UNIT_SYSTEM])
 
+    async def load_translations(_event: Event) -> None:
+        _LOGGER.debug(f"Loading translations for language: {hass.config.language}")
+        await load_state_translations_to_cache(hass, language=hass.config.language)
+
+    hass.bus.async_listen(EVENT_COMPONENT_LOADED, load_translations)
+    hass.bus.async_listen_once(EVENT_HOMEASSISTANT_STARTED, load_translations)
```

Comment:
Why do we need to listen to `EVENT_HOMEASSISTANT_STARTED`?

Reply:
I have added it just as a fallback, it should work without it as well

Reply:
Removed